pkg/logs/internal/status/info.go

```diff
@@ -71,9 +67,9 @@ func (c *CountInfo) Info() []string {
 
 // MappedInfo collects multiple info messages with a unique key
 type MappedInfo struct {
+	mu       sync.Mutex
```

Comment:
Is there a particular reason for renaming it `mu` instead of `lock` previously ? I just find this name less clear